sample_templates/ec2.yml.j2

```diff
@@ -14,6 +14,59 @@ Metrics:
     InstanceId: "{{ instance.id }}"
   Options:
     {#- I'm assuming my tag names are safe to use as metric names here #}
-    Formatter: 'cloudwatch.%(Namespace)s.{{ instance.tags['Name'] }}.%(MetricName)s.%(statistic)s.%(Unit)s'
+    Formatter: 'collectd.{{ instance.tags['Name'] }}.%(MetricName)s.%(statistic)s.%(Unit)s'
     Period: 5
+
+- Namespace: "AWS/EC2"
+  MetricName: "DiskWriteOps"
```

Comment:
As an FYI, similar stats like DiskWriteOps and DiskReadOps are almost always safe to combine, eg:
```
  MetricName:
    - DiskWriteOps
    - DiskReadOps
```

Even stats with different units can be combined this way, leadbutt relies on the unit name returned from CloudWatch when it is not specified in the yaml/config.